testng/src/test/java/io/specto/hoverfly/testng/HoverflyDslMatcherTest.java

```diff
@@ -0,0 +1,336 @@
+package io.specto.hoverfly.testng;
+
+import io.specto.hoverfly.junit.dsl.HttpBodyConverter;
+import io.specto.hoverfly.testng.api.TestNgClassRule;
+import io.specto.hoverfly.testng.models.SimpleBooking;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import java.net.URI;
+import java.time.LocalDate;
+
+import static io.specto.hoverfly.junit.core.SimulationSource.dsl;
+import static io.specto.hoverfly.junit.dsl.HoverflyDsl.service;
+import static io.specto.hoverfly.junit.dsl.HttpBodyConverter.json;
+import static io.specto.hoverfly.junit.dsl.HttpBodyConverter.xml;
+import static io.specto.hoverfly.junit.dsl.ResponseCreators.*;
+import static io.specto.hoverfly.junit.dsl.matchers.HoverflyMatchers.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.springframework.http.HttpStatus.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.http.MediaType.APPLICATION_XML;
+
+@Listeners(HoverflyListener.class)
+public class HoverflyDslMatcherTest {
```

Comment:
I don't think you need to repeat these tests as it's for testing the DSL defined in the root project